src/components/Cluster/ClusterDetail/KeypairCreateModal/__tests__/KeyPairCreateModal.tsx

```diff
@@ -236,22 +225,5 @@ describe('KeyPairCreateModal', () => {
 
     expect(mockActions.clusterCreateKeyPair).toHaveBeenCalled();
     expect(mockActions.clusterLoadKeyPairs).toHaveBeenCalled();
-
-    const kubeconfigContent = getByTitle(/kubeconfig/i).textContent;
-    const copyButton = getByText(/^copy$/i);
-    fireEvent.click(copyButton);
-    expect(setClipboardContentMockFn).toHaveBeenCalledWith(kubeconfigContent);
-    // @ts-ignore
-    useCopyToClipboard.mockReturnValueOnce([true, setClipboardContentMockFn]);
-
-    act(() => {
-      jest.runAllTimers();
-    });
-
-    const copiedSuccessLabel = getByTitle(/content copied to clipboard/i);
-    expect(copiedSuccessLabel).toBeInTheDocument();
-
-    // @ts-ignore
-    useCopyToClipboard.mockClear();
```

Comment:
Removed the part that was testing copying to clipboard, that's already tested for the `FileBlock` component